sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/samples/CreateNewSampleSteps.java

```diff
@@ -722,6 +679,59 @@ public CreateNewSampleSteps(
           softly.assertAll();
         });
 
+    And(
+        "I select Sent dispatched checkbox",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(SAMPLE_SHIPPED_CHECKBOX);
+          webDriverHelpers.clickOnWebElementBySelector(SAMPLE_SHIPPED_CHECKBOX);
+        });
+
+    And(
+        "I select Received checkbox",
+        () -> {
+          webDriverHelpers.waitUntilElementIsVisibleAndClickable(SAMPLE_RECEIVED_CHECKBOX);
+          webDriverHelpers.clickOnWebElementBySelector(SAMPLE_RECEIVED_CHECKBOX);
+        });
+
+    Then(
+        "I check is Sent dispatched Date and Received Date fields required",
+        () -> {
+          webDriverHelpers.waitUntilIdentifiedElementIsPresent(DATE_SAMPLE_SEND);
+
+          webDriverHelpers.waitUntilIdentifiedElementIsPresent(DATE_SAMPLE_RECEIVED);
+        });
+
+    And(
+        "I click Add Pathogen test in Sample creation page in German",
+        () -> webDriverHelpers.clickOnWebElementBySelector(ADD_PATHOGEN_TEST_BUTTON_DE));
+
+    And(
+        "I check DATE AND TIME OF RESULT field",
+        () -> {
+          TimeUnit.SECONDS.sleep(10);
+          webDriverHelpers.getWebElement(DATE_AND_TIME_OF_RESULTS_DE);
+        });
+
+    And(
+        "I click on save sample button",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_SAMPLE_BUTTON);
+        });
+
+    Then(
+        "I check error popup message in German",
```

Comment:
as commented on created method, just get the list of elements, and use assert for this

Reply:
Added assert in checkIsPopupContainsList method.